Editor/AGS.Types/View.cs

```diff
@@ -9,8 +9,7 @@ namespace AGS.Types
     [DefaultProperty("Name")]
     public class View : IToXml, IComparable<View>
     {
-        public delegate void ViewUpdatedHandler(View view);
-        public event ViewUpdatedHandler ViewUpdated;
+        public event Action<View> ViewUpdated;
```

Comment:
I suggest to keep events in interfaces (such as AGS.Types library) unchanged, and later remake them into proper events that take 2 arguments: `(object sender, XEventArgs args)`